routers/api/v1/repo/pull.go

```diff
@@ -210,7 +210,7 @@ func CreatePullRequest(ctx *context.APIContext, form api.CreatePullRequestOption
 		milestoneID = milestone.ID
 	}
 
-	if len(form.Assignee) > 0 {
+	/*if len(form.Assignee) > 0 {
```

Comment:
Do we need this anymore? If not please remove this!

Reply:
No, I simply forgot about it